code/modules/mob/living/simple_animal/friendly/fox.dm

```diff
@@ -22,12 +22,66 @@
 	var/turns_since_scan = 0
 	var/mob/living/movement_target
 
+/mob/living/simple_animal/pet/fox/New()
+	if(!vore_organs.len)
+		var/datum/belly/B = new /datum/belly(src)
```

Comment:
@Arokha to be fair the whole fox file on your codebase is unique to vorestation, it could use the variables in the simple_animal_vr.dm file to define all of this